scripts/runProduction.sh

```diff
@@ -1,5 +1,14 @@
 echo "Running Production Site"
-git clone https://Discordius:$GITHUB_CREDENTIALS_REPO_PAT@github.com/$GITHUB_CREDENTIALS_REPO_NAME.git Credentials 
+
+# lw-look here: you must define GITHUB_CREDENTIALS_REPO_USER in your AWS EBS config
+git clone https://GITHUB_CREDENTIALS_REPO_USER:$GITHUB_CREDENTIALS_REPO_PAT@github.com/$GITHUB_CREDENTIALS_REPO_NAME.git Credentials
```

Comment:
Also I think this is missing a `$`

Reply:
Hah, shoulda read your comments earlier.